src/views/Nft/market/Collection/Items/CollectionNfts.tsx

```diff
@@ -17,33 +22,75 @@ interface CollectionNftsProps {
 
 const CollectionNfts: React.FC<CollectionNftsProps> = ({ collection, sortBy = 'updatedAt' }) => {
   const { address } = collection
+  const [allPancakeBunnyNfts, setAllPancakeBunnyNfts] = useState<NftToken[]>(null)
   const checksummedAddress = getAddress(address)
   const nfts = useNftsFromCollection(checksummedAddress)
   const dispatch = useAppDispatch()
 
+  const isPBCollection = address === pancakeBunniesAddress
+
   useEffect(() => {
     dispatch(fetchNftsFromCollections(checksummedAddress))
   }, [checksummedAddress, dispatch])
 
-  if (!nfts) {
+  useEffect(() => {
+    const fetchPancakeBunnies = async () => {
+      // In order to not define special TS type just for PancakeBunnies display we're hacking a little bit into NftToken type.
```

Comment:
Some TS hacking here since tokenId is required but on this page it is actually not used anywhere.